exporter/newrelicexporter/factory.go

```diff
@@ -30,8 +31,7 @@ func NewFactory() component.ExporterFactory {
 	return exporterhelper.NewFactory(
 		typeStr,
 		createDefaultConfig,
-		exporterhelper.WithTraces(createTraceExporter),
-		exporterhelper.WithMetrics(createMetricsExporter))
```

Comment:
This line does have the effect of enabling the metrics exporter, so we'll likely want to conditionally enable the various exporters.

Reply:
Added `NEW_RELIC_NO_METRIC_SUPPORT` as a means for us to disable metric support for now.